tests/driver_lis2dh12/Makefile

```diff
@@ -1,7 +1,7 @@
 include ../Makefile.tests_common
 
-# as default we use the SPI mode for now, as the I2C mode is not supported, yet
-DRIVER ?= lis2dh12_spi
+# use lis2dh12_spi for SPI-Mode and lis2dh12 for I2C-Mode
+DRIVER ?= lis2dh12
```

Comment:
I would leave the SPI default.

Shouldn't the other variant be called `lis2dh12_i2c`? Or doesn't the comment make sense any more?

Reply:
changed to lis2dh12_i2c 

see [https://github.com/RIOT-OS/RIOT/pull/12290](url)